src/components/authoring/json-editor.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import JSONInput from "react-json-editor-ajrm";
+import locale from "react-json-editor-ajrm/locale/en";
+import * as Ajv from "ajv";
+import schema from "../../glossary-definition-schema";
+
+import * as css from "./json-editor.scss";
+
+interface IProps {
+  placeholder?: object;
```

Comment:
I don't like the name 'placeholder'.  

I get that is the property name used by `react-json-editor-ajrm`  but maybe we could use a more descriptive name in our component. Maybe `glossaryDefinition`?

Reply:
I don't like it either, I've just used it to keep APIs similar.
If it was a controlled component, I'd call this property `value`.
However, it is an uncontrolled one. In this case, I tend to use `initialValue` to stress this fact. 
By controlled and uncontrolled I mean the same thing that is present in other React scenarios, e.g. controlled and uncontrolled text input.
Does `initialValue` work for you?

Reply:
yes, `initialValue ` Sounds good.  Thanks